### PR TITLE
Deployment docs improved

### PR DESCRIPTION
**Pull request by nilminiwso2**
Deployment docs

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -1415,7 +1415,7 @@ nav:
                     - Deploy on Kubernetes: install-and-setup/setup/mi-setup/deployment/kubernetes_deployment.md
                 - Deployment Synchronization: install-and-setup/setup/mi-setup/deployment/deployment_synchronization.md
             - Deploying the Streaming Integrator:
-                - Streaming Integrator Deployment Guide: install-and-setup/setup/si-deployment/deployment-guide.md
+                - Streaming Integrator Deployment - Overview: install-and-setup/setup/si-deployment/deployment-guide.md
```
Comment (samgnaniah) [inline en/mkdocs.yml:1418]: Can we avoid the dash?

File: en/docs/install-and-setup/setup/single-node/configuring-a-single-node.md [hunk 2]
```diff
@@ -107,12 +107,9 @@ In this case, let's use `api.am.wso2.com` as the hostname:
...
 ## Step 6 - Configure API-M Analytics
 
-If you wish to view reports, statistics, and graphs related to the APIs deployed in the WSO2 API Manager, you need to 
-configure API-M Analytics. If not, you can **skip this step**.
+API Manager Analytics is delivered via the API Manager Analytics cloud solution. You need to configure the API-M Gateway needs to publish analytics data into the cloud.
 
-Follow the [Configuring API-M Anlaytics - Quick Setup]({{base_path}}/observe/api-manager-analytics/configuring-apim-analytics/#quick-setup) to configure API-M Analytics in a development setup and, follow 
-[Configuring API-M Analytics - Standard Setup]({{base_path}}/observe/api-manager-analytics/configuring-apim-analytics/#standard-setup) 
-to configure API-M Analytics in a production setup.
+See the instructions on [configuring the synapse Gateway]({{base_path}}/observe/api-manager-analytics/configure-analytics/configure-synapse-gateway) with the cloud-based analytics solution.
```
Comment (samgnaniah) [inline en/docs/install-and-setup/setup/single-node/configuring-a-single-node.md:112]: We don't use the term "synapse gateway" now. @tishan89 should we just say "gateway"?

File: en/docs/install-and-setup/setup/mi-setup/deployment/kubernetes_deployment.md [hunk 3]
```diff
@@ -7,10 +7,10 @@ You can set up a Kubernetes cluster for the Micro Integrator in two ways:
...
+The API operator for Kubernetes provides first-class support for Micro Integrator deployments in the Kubernetes ecosystem. It uses the `Integration` custom resource (`integration_cr.yaml` file) that is available in the Kubernetes module (exported from WSO2 Integration Studio) and deploys the integration in your Kubernetes environment.
+
+See [Deploying Integrations on Kubernetes]({{base_path}}/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integrations/integration-deployments) for more details and instructions.
 
 ## Use Kubernetes resources
 
-!!! Note
-    Work in progress
\ No newline at end of file
+Integration developers can directly generate the kubernetes artifacts that are required for a kubernetes deployment using WSO2 Integration Studio. Once these resources are generated, you can set a Kubernetes deployment by following the Kubernetes documentation.
```
Comment (samgnaniah) [inline en/docs/install-and-setup/setup/mi-setup/deployment/kubernetes_deployment.md:16]:
```suggestion
Integration developers can directly generate the Kubernetes artifacts that are required for a Kubernetes deployment using WSO2 Integration Studio. Once these resources are generated, you can set a Kubernetes deployment by following the Kubernetes documentation.
```

File: en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md [hunk 4]
```diff
@@ -37,10 +37,11 @@ Create an SSL certificate for each of the WSO2 API-M nodes (i.e., Publisher, Dev
 
 When maintaining high availability (HA) in the WSO2 API-M distributed set up, you need to create and import an SSL certificate for each of the WSO2 API-M HA nodes.
 
-
 ## Step 5 - Configure API-M Analytics
 
-If you wish to view reports, statistics, and graphs related to the APIs deployed in the Developer Portal, you need to configure WSO2 API-M Analytics. Follow the [standard setup]({{base_path}}/observe/api-manager-analytics/configuring-apim-analytics/#standard-setup) to configure API-M Analytics in a production setup, and follow the [quick setup]({{base_path}}/observe/api-manager-analytics/configuring-apim-analytics/#quick-setup) to configure API-M Analytics in a development setup.
+API Manager Analytics is delivered via the API Manager Analytics cloud solution. You need to configure the API-M Gateway needs to publish analytics data into the cloud.
```
Comment (samgnaniah) [inline en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md:42]:
```suggestion
API Manager Analytics is delivered via the API Manager Analytics cloud solution. You need to configure the API-M Gateway to publish analytics data into the cloud.
```

File: en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md [hunk 5]
```diff
@@ -37,10 +37,11 @@ Create an SSL certificate for each of the WSO2 API-M nodes (i.e., Publisher, Dev
...
+
+See the instructions on [configuring the synapse Gateway]({{base_path}}/observe/api-manager-analytics/configure-analytics/configure-synapse-gateway) with the cloud-based analytics solution.
```
Comment (samgnaniah) [inline en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md:44]: Let's call it API Manager Gateway.

File: en/docs/install-and-setup/setup/single-node/configuring-a-single-node.md (hunk 2 above)
Comment (tishan89) [reply en/docs/install-and-setup/setup/single-node/configuring-a-single-node.md:112]: Name is still under discussion afaik. Ive asked Sanjeewa and will confirm with u shortly

**Review by samgnaniah — APPROVED** [review]

File: en/docs/install-and-setup/setup/single-node/configuring-a-single-node.md (hunk 2 above)
Comment (tishan89) [reply en/docs/install-and-setup/setup/single-node/configuring-a-single-node.md:112]: Lets use API Gateway

**Pull request merged**